packages/components/src/components/tooltip/tooltip.js

```diff
@@ -101,32 +108,50 @@ class Tooltip extends mixin(
    */
   _hasContextMenu = false;
 
+  /**
+   * The handle for `click` event listener.
+   * Used for "click-outside-tooltip" feature.
+   * @type {Handle}
+   * @private
+   */
+  _handleClickListener;
+
   /**
    * The debounced version of the event handler.
    * @type {Function}
    * @private
    */
   _debouncedHandleClick = debounce(this._handleClick, 200);
 
+  /**
+   * Key codes for allowed keys that will trigger opening a tooltip
+   * @type {Integer[]}
+   * @private
+   */
+  _allowedOpenKeys = [32, 13];
```

Comment:
Would you want to hoist this as a `const` in module top-level? Or in case you want to allow configuration/inheritance, would you want to move this to `this.constructor.options`?